include/drivers/sensor.h

```diff
@@ -80,7 +80,16 @@ enum sensor_channel {
 	/** Ambient temperature in degrees Celsius. */
 	SENSOR_CHAN_AMBIENT_TEMP,
 	/** Pressure in kilopascal. */
-	SENSOR_CHAN_PRESS,
+        SENSOR_CHAN_PRESS,
```

Comment:
use <tab> and not <space> everywhere.
Infact, SENSOR_CHA_PRESS shouldn't be in the commit.